### DISCUSSION TITLE
How to use with Angular' Standalone component

### DISCUSSION THREAD

**Discussion by momoking** [assuncaocharles/ngx-indexed-db#396] [Q&A]
Hi,
I created a Angular 18 project with ngx-indexed-db v19.0.0, it is by default a Standalone component, which means there is no NgModule.

I could not find any guidance on how to make the import work in the standalone mode.

First of all, is standalone mode supported? If so, could you please provide a sample bootstrap code?

Thank you

**Comment by aparzi** [comment]
Hi @momoking, 
with new version 19.2.0 it is now support standalone project.
